src/utils/commandLineBuilder.ts

```diff
@@ -0,0 +1,156 @@
+import { ShellQuotedString, ShellQuoting } from "vscode";
+
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
>Copyright [](start = 3, length = 9)

Move copyright statement to top of file.  Also, where's the MIT license notice as in the other files?